knowledgebase/common-rbac-queries.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Common RBAC queries for ClickHouse Cloud
+description: "Queries to help grant specific permissions to users."
+date: 2023-09-28
+---
+
+# Common RBAC queries for ClickHouse Cloud
+
+## How do I grant the same permissions as the current user to another user?
+
+```sql
+GRANT CURRENT GRANTS ON *.* TO another_user;
+```
+
+## How do I grant a specific permission to a user based on the grants of the current user?
+
+In the below example, `another_user` will be able to perform `SELECT` commands on all of the databases and tables of the current user.
+
+```sql
+GRANT CURRENT GRANTS(SELECT ON *.*) TO another_user;
+```
+
+## How do I grant a specific permission to a user for a specific database based on the grants of the current user?
+
+In the below example, `another_user` will be able to perform `INSERT` commands to all tables in `my_database`.
+
+```sql
+GRANT INSERT ON my_database.* TO another_user;
+```
+
+## How do I grant access to all databases and tables for a specific user?
```

Comment:
I think explanation not correct. It also gives access to all system queries, table functions etc that default user has in cloud. So it is more like "give access to all grants that are available for default user"

Reply:
Thanks, I changed this to: "How do I give access to all grants for a specific user based on the default user?"